setup.cfg

```
@@ -112,7 +112,7 @@ astropy = astropy.cfg, CITATION
 astropy.cosmology = *.ecsv
 astropy.utils.tests = data/.hidden_file.txt
 astropy.tests.figures = *.json
-astropy.wcs = include/*/*.h
+astropy.wcs = include/*/*.h, wcsapi/tests/example_4d_tab.fits
```

Should be something like this to be more consistent with the rest of the package.
```suggestion
astropy.wcs = include/*/*.h
astropy.wcs.tests = data/example_4d_tab.fits
```